### ISSUE TITLE
[BUG] Steam Deck OLED on latest 3.6 firmware , with chiaki-ng 1.9.1 can't see PS5 Pro through PSN 

### ISSUE THREAD

**Issue by L0WR1D3Rz** [streetpea/chiaki-ng#464]
**Describe the bug**
I can remote play without issues in home network. When I connect to another one (e.g. A friend's wifi) my steam deck oled can't see te PS5 pro. Using the official Android app on the same friend's wifi sees the console without a hitch. I tried refreshing the token but that didn't help either.

**Comment by Rahzadan** [comment]
@L0WR1D3Rz Both you and your friend's network have to have uPnP enabled on your routers. I was able to connect to my ps5 pro at home from a public wifi without issue. The public wifi I was connected to must have also had uPnP. Its usually enabled by default in most consumer grade routers.

**Comment by streetpea** [comment]
Is the ps5 pro updated to the latest firmware? Can you enable verbose logging and see what output you get if you launch chiaki-ng from the console 
```
flatpak run io.github.streetpea.Chiaki4deck
```

**Comment by streetpea** [comment]
Also make sure you ps5 pro has a different name than your ps5

**Comment by streetpea** [comment]
@L0WR1D3Rz I just set up my ps5 pro and it works fine via remote connection via psn using chiaki-ng as well using 1.9.1

**Comment by streetpea** [comment]
@L0WR1D3Rz on your home wifi can you turn off discovery and see if the ps5 pro pops up then?

**Comment by L0WR1D3Rz** [comment]
> @L0WR1D3Rz on your home wifi can you turn off discovery and see if the ps5 pro pops up then?

How do I turn off discovery

**Comment by L0WR1D3Rz** [comment]
> Also make sure you ps5 pro has a different name than your ps5

It does. It has different numbers after PS5

**Comment by L0WR1D3Rz** [comment]
> @L0WR1D3Rz Both you and your friend's network have to have uPnP enabled on your routers. I was able to connect to my ps5 pro at home from a public wifi without issue. The public wifi I was connected to must have also had uPnP. Its usually enabled by default in most consumer grade routers.

I can't find a dedicated setting for Upnp in my router admin console. Chiaki used to work fine though a couple of months ago with my base PS5/steamdeck when remote playing. I could play from the other side of the world.

**Comment by L0WR1D3Rz** [comment]
> @L0WR1D3Rz Both you and your friend's network have to have uPnP enabled on your routers. I was able to connect to my ps5 pro at home from a public wifi without issue. The public wifi I was connected to must have also had uPnP. Its usually enabled by default in most consumer grade routers.

Since it works on friend's network using the official app on my mobile, shouldn't this make the Upnp argument irrelevant?

**Comment by streetpea** [comment]
@L0WR1D3Rz click the blue wifi icon to turn it off. Also the remote connection via PSN doesn’t require upnp, it uses upnp first and then STUN

**Comment by L0WR1D3Rz** [comment]
> @L0WR1D3Rz click the blue wifi icon to turn it off. Also the remote connection via PSN doesn’t require upnp, it uses upnp first and then STUN if UPNP doesn’t work just like the official remote play app.

So
1)in home wifi with blue wifi icon at bottom left of chiaki app the console disappears. If I enable it it reappears
2)upnp turned on using my mobile connection Hotspot to test connection through PSN didn't make any difference. It's not visible at all (again tried both with discovery on and off not sure it matters in this case but still tried it). My mobile phone using its 4g connection connects without an issue

**Comment by streetpea** [comment]
@L0WR1D3Rz you don’t need to enable upnp as the official app works with your network which means stun works. Also, even if you did need to enable upnp it has nothing to do with listing the console.

**Comment by streetpea** [comment]
@L0WR1D3Rz please launch via the konsole and post what the console shows with verbose logging enabled. This is before a session is started so there won’t be a session log so you can’t use those.

**Comment by Rahzadan** [comment]
@L0WR1D3Rz This is what it should look like if Chiaki can "see" your PS5 via PSN:

<img width="1436" alt="{F8E11AA8-DE96-4A19-8AEE-B8893115C799}" src="https://github.com/user-attachments/assets/24bac61f-57d5-4bb5-9f57-c081fc019e2c">

If you see nothing, maybe clear your PSN token in the config and refresh it?

**Comment by Rahzadan** [comment]
> @L0WR1D3Rz I just set up my ps5 pro and it works fine via remote connection via psn using chiaki-ng as well using 1.9.1

@streetpea Unrelated, but I can't wait to find out if PS5 Pro can stream in 4K or 120fps.... *fingers crossed, cautiously*

**Comment by streetpea** [comment]
@L0WR1D3Rz In the konsole you should see what the console query for ps5 pro shows. Then also tell me the registered nickname for your ps5 pro is.

**Comment by L0WR1D3Rz** [comment]
> > @L0WR1D3Rz I just set up my ps5 pro and it works fine via remote connection via psn using chiaki-ng as well using 1.9.1
> 
> @streetpea Unrelated, but I can't wait to find out if PS5 Pro can stream in 4K or 120fps.... _fingers crossed, cautiously_

Currently it doesn't. Sony has to enable it.

**Comment by L0WR1D3Rz** [comment]
> @L0WR1D3Rz In the konsole you should see what the console query for ps5 pro shows. Then also tell me the registered nickname for your ps5 pro is.

Ok I'm an idiot..i fixed it. Suggestion for your guide in case it's not clearly stated already, ask users not to change the PS5 nickname through system information after they have registered it with chiaki. When I registered it with Chiaki it was PS5-635 and I renamed it later to PS5 Pro. That's why it wouldn't appear as soon as I changed back to PS5-635 it immediately worked. Apologies for wasting your time streetpea. I don't remember seeing anything in the guide about nickname so if it isn't there maybe add it
